neuro_san/service/agent_main_loop.py

```diff
@@ -158,8 +158,9 @@ def main_loop(self):
             self.port,
             self.http_port,
             self.service_openapi_spec_file,
+            self.request_limit,
             forwarded_request_metadata=self.forwarded_request_metadata)
-        http_server_thread = threading.Thread(target=http_sidecar, daemon=True)
+        http_server_thread = threading.Thread(target=http_sidecar, args=(self.server,), daemon=True)
         http_server_thread.start()
```

Comment:
Pass extra parameter to Tornado server thread - reference to gRPC server we have running.
Use it to stop gRPC server on http service shutdown.

Reply:
tuple is a little wacky there.